integration-test/src/main/java/com/sequenceiq/it/cloudbreak/util/ssh/action/SshJClientActions.java

```diff
@@ -312,19 +326,39 @@ private long executefileListCommand(String instanceIP, String fileListCommand, S
-        String diskMountPointListCmd =
-                "df | grep " + mountDir + " | awk '{print $1,$6}' | " +
-                        "sed -e \"s/\\(.*\\) \\(.*\\)/\\\"\\1\\\":\\\"\\2\\\"/\" | paste -s -d ',' | sed 's/.*/{\\0}/'";
+    public DistroXTestDto checkMeteringStatus(DistroXTestDto testDto, List<InstanceGroupV4Response> instanceGroups, List<String> hostGroupNames) {
+        String meteringStatusCommand = "sudo cdp-doctor metering status --format json";
+        Map<String, Pair<Integer, String>> meteringStatusReportByIp = getDistroXInstanceGroupIps(instanceGroups, hostGroupNames, false).stream()
+                .collect(Collectors.toMap(ip -> ip, ip -> executeSshCommand(ip, meteringStatusCommand)));
+
+        for (Entry<String, Pair<Integer, String>> meteringStatusReport : meteringStatusReportByIp.entrySet()) {
+            List<Integer> heartbeatEventCounts = new Json(meteringStatusReport.getValue().getValue()).getMap().entrySet().stream()
+                    .filter(status -> String.valueOf(status.getKey()).contains("heartbeatEventCount"))
+                    .map(Entry::getValue).collect(Collectors.toList())
+                        .stream()
+                        .map(countObject -> (Integer) countObject)
+                        .collect(Collectors.toList());
+            LOGGER.info(String.format("heartbeatEventCounts: %s", heartbeatEventCounts));
+            Log.log(LOGGER, format(" Found '%s' Metering Heartbeat Events at '%s' instance. ", heartbeatEventCounts, meteringStatusReport.getKey()));
+            if (CollectionUtils.isEmpty(heartbeatEventCounts) || heartbeatEventCounts.contains(0)) {
+                LOGGER.error("Metering Heartbeat Events does NOT generated on '{}' instance!", meteringStatusReport.getKey());
+                throw new TestFailException(String.format("Metering Heartbeat Events does NOT generated on '%s' instance!", meteringStatusReport.getKey()));
+            }
+        }
 
-        Map<String, Pair<Integer, String>> deviceMountPointMappingsByIp = getDistroXInstanceGroupIps(instanceGroups, hostGroupNames, false).stream()
-                .collect(Collectors.toMap(ip -> ip, ip -> executeSshCommand(ip, diskMountPointListCmd)));
-        Integer okStatusCode = 0;
-        deviceMountPointMappingsByIp.forEach((ip, outPair) -> {
-            if (outPair == null || !outPair.getKey().equals(okStatusCode)) {
-                LOGGER.error("Mount point {} should exist on node with IP {}!", mountDir, ip);
-                throw new TestFailException(String.format("Mount point %s should exist on node with IP %s!", mountDir, ip));
+        for (Entry<String, Pair<Integer, String>> meteringStatusReport : meteringStatusReportByIp.entrySet()) {
+            List<String> heartbeatStatusesNotOk = new Json(meteringStatusReport.getValue().getValue()).getMap().entrySet().stream()
+                    .filter(status -> String.valueOf(status.getValue()).contains("NOK"))
```

Comment:
i think it won't be good to check all NOK, as databus unreachable can be valid for a mow-dev run (if databus is shut down or a deploy is happening at that time), but for everything else that can be fine